README.md

```diff
@@ -38,6 +54,12 @@ videojs('video', {}, function() {
 You may also use the Javascript and CSS links from the following to get started:
 [https://cdnjs.com/libraries/videojs-contrib-ads](https://cdnjs.com/libraries/videojs-contrib-ads)
 
+### Using a module system
+
+If you are loading `videojs-contrib-ads` using modules, do this:
+
+TODO
```

Comment:
This TODO still meant to be here?

Reply:
Yeah, I want to link to the example from https://github.com/videojs/videojs-contrib-ads/pull/312 once that is merged. Was planning to do that in rebase. I'll add more info.